docs/getting-started/concepts/permission.md

```diff
@@ -93,29 +93,51 @@ Permission(
 ```
 
 ## Authorizing Feast clients
-**TODO**
-Initial proposals:
-* Kubernetes RBAC:
+If you would like to leverage the permissions' functionality then `auth` config block should be added to feature_store.yaml. Currently, feast supports oidc and kubernetes rbac authentication/authorization. 
+
+The default configuration if you don't mention the auth configuration is no_auth configuration.
+
+* No auth configuration example:
+```yaml
+project: foo
+registry: "registry.db"
+provider: local
+online_store:
+    path: foo
+entity_key_serialization_version: 2
+auth:
+  type: no_auth
```

Comment:
no_auth should be default right ? if it it's default then no need mention this example I think just above description should be fine

Reply:
it is by default but it gives an option for the beginners to explore the other options even though it is by default. I am happy to take it out though.

Reply:
I am going to leave here. this is the place talking about the configuration for `no_auth` also and mentioned it as default. I will remove at other places.